topics/genome-annotation/tutorials/bacterial-genome-annotation/tutorial.md

```diff
@@ -0,0 +1,586 @@
+>    - *"Reference genome to display"*: `Use a genome from history`
+>        - {% icon param-file %} *"Select the reference genome"*: Contig file
+>    - *"Genetic Code"*: `11. The Bacterial, Archaeal and Plant Plastid Code`
+>    - In *"Track Group"*:
+>        - {% icon param-repeat %} *"Insert Track Group"*
+>            - *"Track Category"*: `Bakta`
+>            - In *"Annotation Track"*:
+>                - {% icon param-repeat %} *"Insert Annotation Track"*
+>                    - *"Track Type"*: `GFF/GFF3/BED Features`
+>                        - {% icon param-file %} *"GFF/GFF3/BED Track Data"*: `Annotation_and_sequences` (output of **Bakta** {% icon tool %})
+>                        - *"JBrowse Track Type [Advanced]"*: `Neat Canvas Features`
+>                        - *"Track Visibility"*: `On for new users`
+>        - {% icon param-repeat %} *"Insert Track Group"*
+>            - *"Track Category"*: `Plasmid sequences`
+>            - In *"Annotation Track"*:
+>                - {% icon param-repeat %} *"Insert Annotation Track"*
+>                    - *"Track Type"*: `GFF/GFF3/BED Features`
+>                        - {% icon param-file %} *"GFF/GFF3/BED Track Data"*: `PlasmidFinder GFF`
+>                        - *"JBrowse Track Type [Advanced]"*: `Neat Canvas Features`
+>                        - *"Track Visibility"*: `On for new users`
+>        - {% icon param-repeat %} *"Insert Track Group"*
+>            - *"Track Category"*: `IS elements`
+>            - In *"Annotation Track"*:
+>                - {% icon param-repeat %} *"Insert Annotation Track"*
+>                    - *"Track Type"*: `GFF/GFF3/BED Features`
+>                        - {% icon param-file %} *"GFF/GFF3/BED Track Data"*: GFF output of **ISEScan**
+>                        - *"JBrowse Track Type [Advanced]"*: `Neat Canvas Features`
+>                        - *"Track Visibility"*: `On for new users`
+>
+>        *If integrons are found as **IntegronFinder***
+>
+>        - {% icon param-repeat %} *"Insert Track Group"*
+>            - *"Track Category"*: `Integrons`
+>            - In *"Annotation Track"*:
+>                - {% icon param-repeat %} *"Insert Annotation Track"*
+>                    - *"Track Type"*: `GFF/GFF3/BED Features`
+>                        - {% icon param-file %} *"GFF/GFF3/BED Track Data"*: `IntegronFinder GFF`
+>                        - *"JBrowse Track Type [Advanced]"*: `Neat Canvas Features`
+>                        - *"Track Visibility"*: `On for new users`
+>
+> 3. View the output of JBrowse
+>
+{: .hands_on}
+
+In the output of the JBrowse you can view the genes, IS, plasmid, etc on the contigs. With the search tools you can easily find genes of interest. JBrowse can handle many inputs and can be very useful.
+
+If it takes too long to build the JBrowse instance, you can view an embedded one here. (**Warning**: feature name search will not work.)
```

Comment:
Need a link here?

Reply:
It is embedded directly there. Do we still need a link?

Reply:
no link is fine indeed